### DISCUSSION TITLE
ARtag and Arena Issue

### DISCUSSION THREAD

**Discussion by hinnatariq** [larics/icuas22_competition#55] [Q&A]
Hi @aivano & @fpetric , 

I'm trying to run the simulation with docker using arguments focal with no gpu, it was working fine before with the setup. But suddenly the simulation environment looks like this now, no artag, some obstacles missing and a side wall disappeared. But I don't get any error and at the same time can't execute the code since the artag disappeared. Can anyone clarify the reason for the issue? Since can't work on this anymore. I tried re-installing the setup. But the same issue persists. Is anyone else also facing such issue, and how to resolve?

![279285894_397241348670713_5898639065922799971_n](https://user-images.githubusercontent.com/104435984/165298471-cd035df1-bf1d-45f1-8297-a2b8f746853c.jpg)

**Comment by fpetric** [comment]
Can you confirm you updated both the main competition package and also larics_gazebo_worlds?

**Reply by hinnatariq** [reply]
Yeah, I even tried doing this 2 to 3 times

**Reply by fpetric** [reply]
Can you post which OS, ROS and Gazebo you are using? This looks rather strange to me

**Reply by hinnatariq** [reply]
I am working on Ubuntu 20 focal fossa, ROS Noetic, Gazebo 11.

**Reply by lmark1** [reply]
A freshly built focal-nogpu Docker image is working as expected on my laptop with no GPU. I am not sure what the problem is.

**Comment by fpetric** [comment]
Hi @hinnatariq, 

we have added a few new arenas and reorganized texture files. Can you pull latest updates and try launching the challenge again?

**Reply by hinnatariq** [reply]
Yes I pulled latest updates but still the issue persists. It was working fine for some days suddenly since yesterday this issue occured. And its not even giving any errors 😔

**Reply by hinnatariq** [reply]
@fpetric @lmark1 the issue still stands on my system. Part of the wall and obstacles are missing to test my codes and work on this. Even after new installation its showing like this, only a partial arena not complete. Please guide if possible?

**Reply by fpetric** [reply]
We have tried both noetic and melodic versions, with and without GPUs and we cannot reproduce the issue you are having. Maybe you can checkout an older version of all packages and see if that works? Maybe that would tell us if the issue is caused by us updating the textures or something on your end.
The competition repo before adding textures is on commit 4338ebeec8e108ce445b20f5dcc41822f3a7dc2c, larics_gazebo_worlds was on aa55debef1aa9c73d3216d1d71be9ece964582fe.
